### PR TITLE
service implemented

### PR DESCRIPTION
**Pull request by arceruiz**

### CODE REVIEW

File: internal/config/config.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	Cfg Config
+)
+
+type Config struct {
+	Token struct {
+		Key string `cfg:"key"`
+	} `cfg:"token"`
+	Server struct {
+		Port         string `cfg:"port"`
+		CustomerPort int    `cfg:"customerPort"`
```
Comment (mauriciodm1998) [inline internal/config/config.go]: customerPort?

File: internal/config/config.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+server:
+  customerPort: 3002
+  port: 3001
```
Comment (mauriciodm1998) [inline internal/config/config.yaml]: that's the same port of the payment service

File: internal/config/config.go (hunk 1 above)
Comment (arceruiz) [reply internal/config/config.go]: done

File: internal/config/config.yaml (hunk 2 above)
Comment (arceruiz) [reply internal/config/config.yaml]: done

**Pull request merged**